nipype/interfaces/mrtrix3/preprocess.py

```diff
@@ -40,10 +40,13 @@ class ResponseSDInputSpec(MRTrix3BaseInputSpec):
         argstr='%s', position=-1, desc='output CSF response text file')
     in_mask = File(
         exists=True, argstr='-mask %s', desc='provide initial mask image')
-    max_sh = traits.Int(
-        8, usedefault=True,
-        argstr='-lmax %d',
-        desc='maximum harmonic degree of response function')
+    max_sh = traits.Either(
+        traits.Int(8), traits.List(traits.Int()),
+        default=8,
+        usedefault=True,
+        argstr='-lmax %s',
+        desc=('maximum harmonic degree of response function - single value for '
+              'single-shell response, list for multi-shell response'))
```

Comment:
Can this be a `MultiObject`?

```Python
max_sh = InputMultiObject(traits.Int, default=8, usedefault=True, argstr='-lmax %d', sep=',')
```

I think you can keep the same doctest.

Reply:
This was my inital thought, but it looks like `InputMultiPath/Object` doesn't allow for defaults
```
In [2]: x.inputs.max_sh
Out[2]: []
```